mmedit/models/base_models/one_stage.py

```diff
@@ -99,7 +100,12 @@ def __init__(self,
 
         self.disc_step_count = 0
 
-    def forward(self, inputs, data_samples, mode='tensor'):
+    def forward(
+        self,
+        inputs: torch.Tensor,
+        data_samples: Optional[SampleList],
+        mode: str = 'tensor'
+    ) -> Union[Tuple[torch.Tensor, torch.Tensor], SampleList]:
```

Comment:
Referring to the docstring, the return type seems to be `Union[dict, torch.Tensor, Tuple[torch.Tensor], SampleList]`.
I suggest defining an alias for this return type at the start of this file, e.g.,
```python
import xxx as xxx

FORWARD_RETURN_TYPE = Union[dict, torch.Tensor, Tuple[torch.Tensor], SampleList]



def forward(self, xxx) -> FORWARD_RETURN_TYPE:
    ...
```